spec/models/period_contribution_calculator_spec.rb

```diff
@@ -108,5 +108,43 @@
         expect(period_contribution.tax_relief).to eq(1.72)
       end
     end
+
+    context 'when employee is contributing more than 40,000 per year' do
+      let(:eligible_salary) { 80_000 }
+      let(:employee_percent) { 60 }
+      let(:employer_percent) { 1 }
+
+      context 'when yearly frequency' do
+        let(:salary_frequency) { 1 }
+
+        it 'returns employee tax relief' do
+          expect(period_contribution.tax_relief).to eq(8000)
+        end
+      end
+
+      context 'when monthly frequency' do
```

Comment:
In some way, tests should be verbose in order to express concisely the contexts. Trying to dry the specs it will make difficult to understand what those about.